cmd/nri-kubernetes/main.go

```diff
@@ -73,19 +74,31 @@ func main() {
 		}
 	}
 
-	iw, err := integration.NewWrapper(
+	integrationOptions := []integration.OptionFunc{
 		integration.WithLogger(logger),
 		integration.WithMetadata(integration.Metadata{
 			Name:    integrationName,
 			Version: integrationVersion,
 		}),
-	)
+	}
+
+	switch c.Sink.Type {
+	case config.SinkTypeHTTP:
+		integrationOptions = append(integrationOptions, integration.WithHTTPSink(c.Sink.HTTP))
+	case config.SinkTypeStdout:
+		logger.Warn("Sinking metrics to stdout")
```

Comment:
should we specify that this is no-op since the dafault is alreadty stdOUT?

Reply:
Fair enough :)